state/event_table.go

```diff
@@ -319,7 +317,7 @@ func (t *EventTable) LatestEventInRooms(txn *sqlx.Tx, roomIDs []string, highestN
 	err = txn.Select(
 		&events,
 		`SELECT event_nid, room_id, event_replaces_nid, before_state_snapshot_id, event_type, state_key, event FROM syncv3_events
-		WHERE event_nid IN (SELECT max(event_nid) FROM syncv3_events WHERE event_nid <= $1 AND room_id = ANY($2) GROUP BY room_id)`,
+		WHERE event_nid IN (SELECT MAX(event_nid) FROM syncv3_events WHERE event_nid <= $1 AND room_id = ANY($2) GROUP BY room_id)`,
```

Comment:
I'm guessing this is some linter doing this, or just your OCD? ;)

Reply:
This is goland. I made the mistake of teaching it how I liked SQL to be formatted. Let me unteach it.

Reply:
![image](https://github.com/matrix-org/sliding-sync/assets/8614563/7d1f62c2-b3ab-4f56-b24f-28e0153baacb)
this is presumably the culprit

Reply:
3e02510